list.go

```diff
@@ -0,0 +1,52 @@
+package gotwtr
+
+type ListField string
+
+const (
+	ListFieldCreatedAt   ListField = "created_at"
+	ListFieldPrivate     ListField = "private"
+	ListFollowerCount    ListField = "follower_count"
+	ListMemberCount      ListField = "member_count"
+	ListOwnerID          ListField = "owner_id"
+	ListFieldDescription ListField = "description"
+)
+
+type List struct {
+	ID            string `json:"id"`
+	Name          string `json:"name"`
+	CreatedAt     string `json:"created_at,omitempty"`
+	Private       bool   `json:"private,omitempty"`
+	FollowerCount int    `json:"follower_count,omitempty"`
+	MemberCount   int    `json:"member_count,omitempty"`
+	OwnerID       string `json:"owner_id,omitempty"`
+	Description   string `json:"description,omitempty"`
+}
+
+type ListIncludes struct {
+	Users []*User
+}
+
+type ListMeta struct {
+	ResultCount   int    `json:"result_count"`
+	PreviousToken string `json:"previous_token,omitempty"`
+	NextToken     string `json:"next_token,omitempty"`
+}
+
+type OwnedListsLookUpByIDResponse struct {
+	Lists    []*List             `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
+	Errors   []*APIResponseError `json:"errors,omitempty"`
+	Meta     *ListMeta           `json:"meta"`
+	Title    string              `json:"title,omitempty"`
+	Detail   string              `json:"detail,omitempty"`
+	Type     string              `json:"type,omitempty"`
```

Comment:
Please tell me reproduce way to display these.

Reply:
Please running this command, invalid request error return this struct. 
`curl "https://api.twitter.com/2/users/111111111111111111111111/owned_lists" -H "Authorization: Bearer $token"`

Reply:
ty